inspire/base/static/js/settings.js

```diff
@@ -100,6 +101,10 @@ require.config({
       deps: ["jquery"],
       exports: "$.fn.tokenInput"
     },
+    "dropdown-menu": {
+      deps: ["jquery"],
+      exports: "$.fn.menuAim"
```

Comment:
So in the end it will be like: 
"jQuery-menu-aim": {
      deps: ["jquery"],
      exports: "$.fn.menuAim"
    },
?